api/entity_config/config.go

```diff
@@ -12,6 +12,12 @@ func ConfigValue(network *net.Network, application, entity, config string) (stri
 	if nil != err {
 		return "", err
 	}
+
+	var value string;
+	err = json.Unmarshal(bytes, &value)
+	if nil == err {
```

Comment:
Really both line 19 and 21 are happy paths - 19 is when you can serialise the JSON config value to a string, ("apache"), and 21 when it's more complex, in which case we just return the JSON text. 

I could do it as 
```go
	if nil != err {
		return string(bytes), nil
	}
       return value, nil
```

if you think that is better style?  Or maybe better with an explicit 'else'?